src/libsync/owncloudpropagator.cpp

```diff
@@ -796,16 +796,16 @@ bool PropagatorCompositeJob::scheduleSelfOrChild()
     }
 
     // Ask all the running composite jobs if they have something new to schedule.
-    for (int i = 0; i < _runningJobs.size(); ++i) {
-        ASSERT(_runningJobs.at(i)->_state == Running);
+    for (auto runningJob : qAsConst(_runningJobs)) {
```

Comment:
Why not by const-reference here?

Reply:
Because this is a scalar type, there's nothing to gain from the const-ref.

Reply:
If that was the case, I wouldn't have asked :-) But `PropagatorJob` doesn't look very scalar to me.

Reply:
`PropagatorJob *` on the other hand. ;-)

Reply:
....looks very scalar. Good :) Acknowledged.